extensions/sql-database-projects/package.json

```diff
@@ -148,11 +150,22 @@
         "category": "%sqlDatabaseProjects.displayName%",
         "icon": "images/databaseProjectToolbar.svg"
       },
+      {
+        "command": "sqlDatabaseProjects.updateProjectFromDatabase",
+        "title": "%sqlDatabaseProjects.updateProjectFromDatabase%",
+        "category": "%sqlDatabaseProjects.displayName%",
+        "icon": "images/databaseProjectToolbar.svg"
```

Comment:
can you add a screenshot where this icon is used?

Reply:
Added (database dashboard icon)

Reply:
the toolbar is starting to look a little crowded with the long names of "Create project from database" and "Update project from database." How much do we think people will use this from the server/database toolbar?